### PR TITLE
Add the 2201.3.0 announcement blog

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the 2201.3.0 announcement blog.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Runtime updates
+
+- The configuration of `nil`, `json`, and array of table values are now supported by configurable variables through the TOML syntax. It is also improved to add filler values when configuring array values
+- The `fromJsonWithType` and `cloneWithType` functions of the `lang.value` module are improved to return the value with the first matching type of the union members when cloning with a union type if multiple matching types are found. Previously, those were panicked with ambiguous errors
+
+ ## Standard Library updates
+
+- Support was added in the `graphql` module for disabling introspection queries, and also for GraphQL interfaces including interfaces implementing interfaces 
+- The `graphql` module is improved with added support for the service-level interceptor execution
```
Comment (anupama-pathirage) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md]: Wdyt about rewording above two points as follows for clarity.
```
- Disabling introspection queries support and GraphQL interfaces support is added in the `graphql` module. Also, `graphql` module is improved with the support for the service-level interceptor execution.
```

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- The Swan Lake Update 3 release allows you to use any expression as the default value of a function type parameter.
+- There were also a few improvements to working with optional fields.
+- An optional terminating semicolon is allowed now for module-level declarations.
+
+## Runtime updates
+
+- The configuration of `nil`, `json`, and array of table values are now supported by configurable variables through the TOML syntax. It is also improved to add filler values when configuring array values.
+- The `fromJsonWithType` and `cloneWithType` functions of the `lang.value` module are improved to return the value with the first matching type of the union members when cloning with a union type if multiple matching types are found. Previously, those were panicked with ambiguous errors.
+
+ ## Standard Library updates
```
Comment (anupama-pathirage) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:30]:
```suggestion
## Standard library updates
```

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are happy to announce a new release, [Ballerina 2201.3.0 (Swan Lake Update 3)](https://ballerina.io/downloads/), the third update release of Ballerina Swan Lake.
+
+The main highlight of this release is the support for generating native executables with GraalVM. This introduces a new `bal build --native` CLI command to generate the GraalVM native executable and the  `bal build --native --cloud=docker` CLI command for the native executable Docker image creation for cloud-enabled projects. For more information, see [Build a native executable](https://ballerina.io/learn/build-a-native-executable/).
```
Comment (anupama-pathirage) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:15]:
```suggestion
The main highlight of this release is the support for generating native executables with GraalVM. This introduces a new `bal build --native` CLI command to generate the GraalVM native executable, and the  `bal build --native --cloud=docker` CLI command for the native executable Docker image creation for cloud-enabled projects. For more information, see [Build a native executable](https://ballerina.io/learn/build-a-native-executable/).
```

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md (hunk 2 above)
Comment (praneesha) [reply _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:30]: `Standard Library` is a pronoun and  both first letters should be uppercase.

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md (hunk 1 above)
Comment (praneesha) [reply _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md]: Updated.

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
```
Comment (sameerajayasoma) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:15]: It's wrong to say `CLI command` here. Here we are introducing a new option, `--native`, to the `bal build` command. 

When you combine `--native` and `--cloud=docker` or `--cloud=k8s`, we build a Docker image with the native executable.  

You can adjust the above statement according to the style we've used in the blog.

**Review by sameerajayasoma — CHANGES_REQUESTED** [review]

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md (hunk 4 above)
Comment (sameerajayasoma) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:15]: Do we need to say that the GraalVM support is experimental in Update3? @shafreenAnfar

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md (hunk 2 above)
Comment (anupama-pathirage) [reply _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:30]: Check the release note, it seems it has other way.

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md (hunk 4 above)
Comment (praneesha) [reply _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md]: Updated.

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are happy to announce a new release, [Ballerina 2201.3.0 (Swan Lake Update 3)](https://ballerina.io/downloads/), the third update release of Ballerina Swan Lake.
+
+The main highlight of this release is the support for generating native executables with GraalVM. This introduces a new `--native` option to the `bal build` CLI command (i.e., `bal build --native`) to generate the GraalVM native executable. Also, when `--native` and `--cloud=docker` or `--cloud=k8s` can be combined to build a Docker image with the native executable for cloud-enabled projects. For more information, see [Build a native executable](https://ballerina.io/learn/build-a-native-executable/).
```
Comment (sameerajayasoma) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:15]: In the following sentence, the usage of `when` doesn't make sense right?
----
Also, when `--native` and `--cloud=docker` or `--cloud=k8s` can be combined to build a Docker image with the native executable for cloud-enabled projects.
----

Comment (praneesha) [reply _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md]: Fixed the typo.

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are happy to announce a new release, [Ballerina 2201.3.0 (Swan Lake Update 3)](https://ballerina.io/downloads/), the third update release of Ballerina Swan Lake.
+
+The main highlight of this release is the support for generating native executables with GraalVM. This introduces a new `--native` option to the `bal build` CLI command (i.e., `bal build --native`) to generate the GraalVM native executable. Also, the `--native` and `--cloud=docker` or `--cloud=k8s` can be combined to build a Docker image with the native executable for cloud-enabled projects. For more information, see [Build a native executable](https://ballerina.io/learn/build-a-native-executable/).
```
Comment (sameerajayasoma) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:15]:
```suggestion
The main highlight of this release is the support for generating native executables with GraalVM. This introduces a new `--native` option to the `bal build` CLI command (i.e., `bal build --native`) to generate the GraalVM native executable. Also, the `--native` and `--cloud=docker` or `--cloud=k8s` can be combined to build a Docker image with the native executable. For more information, see [Build a native executable](https://ballerina.io/learn/build-a-native-executable/).
```

**Review by sameerajayasoma — CHANGES_REQUESTED** [review]
I just had another look at this blog. It can be improved a lot. It's too wordy. 

e.g. `
The other significant new additions and improvements belonging to multiple areas that are done in this release include the below.`

Just give it a try using Grammarly. You'll see what I am talking about!

**Comment by praneesha** [comment]
> I just had another look at this blog. It can be improved a lot, and it's too wordy.
> 
> e.g. ` The other significant new additions and improvements belonging to multiple areas that are done in this release include the below.`
> 
> Just give it a try using Grammarly. You'll see what I am talking about!

Improved.

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are happy to announce a new release, [Ballerina 2201.3.0 (Swan Lake Update 3)](https://ballerina.io/downloads/), the third update release of Ballerina Swan Lake.
+
+The main highlight of this release is the support for generating native executables with GraalVM. This introduces a new `--native` option to the `bal build` CLI command (i.e., `bal build --native`) to generate the GraalVM native executable. Also, the `--native` and `--cloud=docker` or `--cloud=k8s` can be combined to build a Docker image with the native executable. For more information, see [Build a native executable](https://ballerina.io/learn/build-a-native-executable/).
```
Comment (sameerajayasoma) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:15]:
```suggestion
The main highlight of this release is the support for generating native executables with GraalVM. This introduces a new `--native` option to the `bal build` CLI command (i.e., `bal build --native`) to generate the GraalVM native executable. Also, the `--native` and `--cloud=docker` or `--cloud=k8s` can be combined to build a Docker image with the native executable. See [Build a native executable](https://ballerina.io/learn/build-a-native-executable/).
```

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+The other significant new additions and improvements belonging to multiple areas that are done in this release include the below.
```
Comment (sameerajayasoma) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:17]:
```suggestion
The rest of the significant additions and improvements are described below.
```

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+## Developer tools updates
+
+- The `Run-In-Terminal` feature was introduced to the debugger to accommodate Ballerina programs that take user inputs during the launch mode.
+- The GraphQL tool was improved by introducing single client generation for single schema definitions and, enabling the GraphQL client config generation to support runtime configurability.
+- Changed the default behavior of the OpenAPI client generation to generate resource methods instead of remote methods.
+
+For a detailed list of what’s new in Swan Lake Update 3, see the [release note](https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.3.0).
+
+We want to extend our sincerest thanks to the Ballerina community for your valuable feedback and support in making Ballerina what it is today. We hope you enjoy this release.
```
Comment (sameerajayasoma) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:45]:
```suggestion
We extend our sincerest thanks to the Ballerina community for your valuable feedback and support in making Ballerina what it is today. We hope you enjoy this release.
```

File: _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+## Compiler updates
+
+- The Swan Lake Update 3 release allows you to use any expression as the default value of a function type parameter.
+- There were also a few improvements to working with optional fields.
+- An optional terminating semicolon is allowed now for module-level declarations.
+
+## Runtime updates
+
+- The configuration of nil, JSON, and array of table values are now supported by configurable variables through the TOML syntax. It is also improved to add filler values when configuring array values.
+- The `fromJsonWithType` and `cloneWithType` functions of the `lang.value` module are improved to return the value with the first matching type of the union members when cloning with a union type if multiple matching types are found. Previously, those were panicked with ambiguous errors.
```
Comment (sameerajayasoma) [inline _posts/2022-12-05-announcing-ballerina-2201.3.0-swan-lake-update-3.md:28]:
```suggestion
- The `fromJsonWithType` and `cloneWithType` functions of the `ballerina/lang.value` module are improved to return the value with the first matching type of the union members when cloning with a union type if multiple matching types are found. Previously, those were panicked with ambiguous errors.
```

**Review by sameerajayasoma — CHANGES_REQUESTED** [review]
@praneesha See whether these changes are okay.
